OpenRA.Mods.Common/Traits/Air/FallsToEarth.cs

```diff
@@ -19,10 +19,17 @@ namespace OpenRA.Mods.Common.Traits
 	public class FallsToEarthInfo : ITraitInfo, IRulesetLoaded, Requires<AircraftInfo>
 	{
 		[WeaponReference]
+		[Desc("Explosion weapon that triggers when hitting ground.")]
 		public readonly string Explosion = "UnitExplode";
 
-		public readonly bool Spins = true;
+		[Desc("How fast the tumbling aircraft spins (at most). Negative values default to 'unlimited.",
```

Comment:
Typo: `'unlimited.'.

Minor nit, but since this needs to be changed anyway: putting the value in quotes IMO feels like it is implying that 'unlimited' is a valid value.

How about
`[Desc("Limit the maximum spin (in facing units per tick) that can be achieved while crashing. 0 disables spinning. Negative values imply no limit.")]`?